internal/core/software_renderer.rs

```diff
@@ -318,7 +319,8 @@ fn render_window_frame_by_line<const MAX_BUFFER_AGE: usize>(
             dirty_region.min_x() as usize..dirty_region.max_x() as usize,
             |line_buffer| {
                 let offset = dirty_region.min_x() as usize;
-                TargetPixel::blend_slice(line_buffer, background.into());
+                // FIXME gradient
+                TargetPixel::blend_slice(line_buffer, background.color().into());
```

Comment:
Micro optimization. Would be nice to not call background.color().into()  on every line.  In fact, the PremultipliedRgbaColor color could be cached outside of the `while`   (but maybe the compiler is able to do that anyway?)

Reply:
I have done this optimization.